src/components/AutoCompleteInput.js

```diff
@@ -115,22 +118,16 @@ export default class AutoCompleteInput extends React.Component<AutoCompleteInput
               error: '',
               open,
             });
-          }).catch((error) => {
+          } else if (errorString) {
             this.setState({
               isLoading: false,
               suggestions: [],
-              error,
-              open: error.length > 0,
+              error: errorString,
+              open: errorString.length > 0,
             });
-          });
-        }, (error) => {
-          this.setState({
-            isLoading: false,
-            suggestions: [],
-            error,
-            open: error.length > 0,
-          });
-        });
+          }
+        };
+        FetchUtils.fetch(`${uri}?term=${encodedValue}`, null, callback, 'GET', 'An error occured while looking for suggestions: ');
```

Comment:
Since the error message might occur as-is, I'd change it to end with a period rather than the colon, and no trailing space since the FetchUtils class adds that.